bdsg/include/bdsg/snarl_distance_index.hpp

```diff
@@ -8,11 +8,45 @@
 #include <handlegraph/algorithms/dijkstra.hpp>
 #include <handlegraph/util.hpp>
 #include <handlegraph/trivially_serializable.hpp>
+#include <handlegraph/algorithms/dijkstra.hpp>
 #include <bdsg/internal/mapped_structs.hpp>
 #include <string>
+#include <numeric>
 #include <arpa/inet.h>
+#include <jansson.h>
```

Comment:
I don't think you actually use any Jansson types in this header. In that case, it would be better to put the include in the CPP file.

Otherwise, anyone who includes the distance index header also has to read the Jansson header, and the compiler will waste a bunch of time reading the Jansson header when it isn't going to be needed to build that `.o` file.